countries.css

```diff
@@ -0,0 +1,54 @@
+@import url(index.css);
+
+body {
+    background-color: var(--primary-color) ;
+  display: grid;
+   align-items: center;
+   place-items: center;
+
+}
+
+.main-container {
+    width: 500px;
+    display: grid;
+    place-items: center;
+    align-items: center;
+}
+
+.main-container .scoreboard {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 300px;
+    color: var(--secondary-color);
+    padding: 0px;
+    margin: 0px;
+    font-size: 1.3rem;
+    white-space: nowrap;

```

Comment:
I don't understand the usage of `white-space: nowrap` here. The text might continue to run beyond the users' view area, a horizontal scroll-bar might appear. I think you can manage whatever you're trying to do without this as well.